spec/spec_helper.rb

```diff
@@ -40,7 +40,7 @@
 
   # This option will default to `:apply_to_host_groups` in RSpec 4 (and will
   # have no way to turn it off -- the option exists only for backwards
-  # compatibility in RSpec 3). It causes shared context metadata to be
+  # compatibility in RSpec 3). It causes helpers context metadata to be
```

Comment:
This looks like a search/replace that shouldn't have happened.

Reply:
Thats a weird change and I dont remember making it or doing any large search/replace, so I've undone the change. Sorry about that.

Reply:
Wierdly I'm not seeing this in the repo: 

https://github.com/ubiquitypress/hyku_addons/blob/Feature/RisDocumentType-chris/spec/spec_helper.rb#L43

Seems to have been changed. Not sure why this is shown as different from what was commited.

Reply:
Looks like that link is for the branch for #30.  I'm seeing it in the `more_work_types` branch: https://github.com/ubiquitypress/hyku_addons/blob/more_work_types/spec/spec_helper.rb#L43

Reply:
Strange. i'd not merged that branch. :-/